numpy/__init__.py

```diff
@@ -357,6 +357,26 @@ def _mac_os_check():
                 raise RuntimeError(msg)
     del _mac_os_check
 
+    def _win_os_check():
+        """
+        Quick Sanity check for Windows OS: look for fmod bug issue 16744.
+        """
+        try:
+            a = np.arange(13 * 13, dtype=np.float64).reshape(13, 13)
+            a = a % 17  # calls fmod
+            np.linalg.eig(a)
```

Comment:
whoops, we already imported numpy

```suggestion
            a = arange(13 * 13, dtype= float64).reshape(13, 13)
            a = a % 17  # calls fmod
            linalg.eig(a)
```